playwright/_impl/_page.py

```diff
@@ -1029,6 +1046,8 @@ async def pdf(
         preferCSSPageSize: bool = None,
         margin: PdfMargins = None,
         path: Union[str, Path] = None,
+        outline: bool = None,
```

Comment:
Shouldn't these be `outline: Optional[bool] = None` etc.

Reply:
Please let me know if I am missing something.

Reply:
We use `strictParameterNoneValue = false` hence changing a single parameter won't make a big difference there.

Reply:
https://github.com/microsoft/playwright-python/blob/976f04fb4d56d1497fd85e0c5199a9daeed4a138/pyproject.toml#L44